notification/src/handler/notification/notification.handler.js

```diff
@@ -270,6 +277,29 @@ function getChangeTransactionStateUpdateAction(
   }
 }
 
+function getChangeTransactionTimestampUpdateAction(
+  transactionId,
+  transactionEventDate
+) {
+  let transactionTimestamp = ''
+  try {
+    // Assume transactionEventDate should be in correct format (e.g. '2019-01-30T18:16:22+01:00')
+    const eventDateMilliSecondsStr = Date.parse(transactionEventDate)
+    const transactionDate = new Date()
+    transactionDate.setTime(eventDateMilliSecondsStr)
+    transactionTimestamp = transactionDate.toISOString()
+  } catch (err) {
```

Comment:
We just cache and log nothing? In case the Adyen format is always wrong we would never be able to use Adyen date and would not even notice it?

Reply:
Add error logging when conversion failed
https://github.com/commercetools/commercetools-adyen-integration/pull/826/commits/a0187be017ff3979aba687198953b0738085aea0